packages/reactive-element/src/reactive-element.ts

```diff
@@ -728,16 +729,24 @@ export abstract class ReactiveElement
     key: string | symbol,
     options: PropertyDeclaration
   ): PropertyDescriptor | undefined {
+    const {get, set} = getOwnPropertyDescriptor(this.prototype, name) ?? {};
     return {
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       get(): any {
-        return (this as {[key: string]: unknown})[key as string];
+        return get === undefined
```

Comment:
actually... I want to change this so that the conditional is outside of the accessors and run only once